.eslintrc.json

```diff
@@ -24,12 +24,18 @@
     {
       "files": ["*.ts", "*.tsx"],
       "extends": ["plugin:@nx/typescript"],
-      "rules": {}
+      "rules": {
+        "@typescript-eslint/no-extra-semi": "error",
```

Comment:
Why was this needed? The ts version of the no extra semi is a fairly expensive rule

Reply:
I have no idea. This was done by the migrations when updating to the latest Nx. 

I'll revert.